templates/app_page.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+
+{% block content%}
+    <div class="content">
+        <!-- TODO: change href -->
+        <div class="kbcb-back-link"><a href="{{url_for('get_apps')}}"><i class="fa fa-chevron-left"></i> back to the Catalog</a></div>
+        <!-- {{ app }} -->
+        <div class="app-header">
+            <div class="app-header-icon">
+                {% if app.icon %}
+                <img class="icon-img-l" src="https://kbase.us/services/narrative_method_store/{{ app.icon.url }}" >
+                {% else %}
+                <i class="far fa-question-circle"></i>
+                <img class="icon-img-l" src="https://cdn3.iconfinder.com/data/icons/trico-rounded-squares/24/Rounded-Square-Question-512.png">
```

Comment:
Why not use a font-awesome icon? Or glyphicon? Those are already included in the stack.